resources/install-letsencrypt-cert.sh

```diff
@@ -31,23 +31,23 @@ if [ ! -x "$CERTBOT" ] ; then
         apt-get update
         apt-get -y install certbot
     elif [ "$DISTRO" = "Ubuntu" ]; then
-        if [ "$DISTRO_VERSION" = "20.04" ] || [ "$DISTRO_VERSION" = "19.10" ]; then
-                apt-get update
-                apt-get -y install software-properties-common
-                add-apt-repository -y universe
-                apt-get update
-                apt-get -y install certbot
-        elif [ "$DISTRO_VERSION" = "18.04" ]; then
-                apt-get update
-                apt-get -y install software-properties-common
-                add-apt-repository -y universe
-                add-apt-repository -y ppa:certbot/certbot
-                apt-get update
-                apt-get -y install certbot
+        if [ "$DISTRO_VERSION" = "18.04" ]; then
+            apt-get update
+            apt-get -y install software-properties-common
+            add-apt-repository -y universe
+            add-apt-repository -y ppa:certbot/certbot
+            apt-get update
```

Comment:
Most of this is the same in both branches. Is this equivalent?
```
apt-get update
apt-get -y install software-properties-common
add-apt-repository -y universe
if [ "$DISTRO_VERSION" = "18.04" ]; then
  add-apt-repository -y ppa:certbot/certbot
fi  
apt-get update
apt-get -y install certbot
```

Reply:
;)

Reply:
What about now?